packages/ui/src/components/ConfirmModal/index.tsx

```diff
@@ -37,6 +37,14 @@ const ConfirmModal = ({
       overlayClassName={classNames(modalStyles.overlay, styles.overlay)}
       parentSelector={() => document.querySelector('main') ?? document.body}
       ariaHideApp={false}
+      onAfterOpen={() => {
+        /* eslint-disable @silverhand/fp/no-mutation */
+        document.body.style.overflow = 'hidden';
+      }}
+      onAfterClose={() => {
+        document.body.style.overflow = 'unset';
+        /* eslint-enable @silverhand/fp/no-mutation */
+      }}
```

Comment:
Is it possible to use CSS? For example:
```typescript
document.body.classList.add(className);
```

Reply:
Or add some comments?

Reply:
Adding classname would require a global classname defined, not sure which one is more straightforward?

Reply:
I'll go with the classname then

Reply:
@wangsijie @demonzoo updated PTAL